docs/README.md

```diff
@@ -27,6 +27,9 @@ user:
 news:
   title: Events & News
   content:
+  - header: SkyWalking Cloud on Kubernetes 0.1.0 is released
+    content: OAPServer CRDs and controller
+    time: Nov. 16th, 2020
```

Comment:
Seems to have missed remove expired event :)

Reply:
done